src/dict.c

```diff
@@ -74,12 +74,19 @@ struct dictEntry {
                                  * by dictType's dictEntryMetadataBytes(). */
 };
 
+typedef struct {
+    void *key;
+    dictEntry *next;
+} dictEntry_no_value;
```

Comment:
What about `dictNoValueEntry`? snake case and camel case mix look odd.

Reply:
You're right, it is very odd. :-) I'll change to camel case.